src/RoadRegistry.BackOffice.UI/src/modules/activity/views/Activities.vue

```diff
@@ -172,7 +181,9 @@
         <vl-column>
           <div v-vl-flex v-vl-flex:align-center>
             <vl-button mod-loading v-if="pagination.isLoading"></vl-button>
-            <vl-button v-else-if="activities.length >= pagination.pageSize" v-on:click="loadNextPage()">Meer ...</vl-button>
+            <vl-button v-else-if="activities.length >= pagination.pageSize" v-on:click="loadNextPage()"
+              >Meer ...</vl-button
```

Comment:
can `</v1-button` be on the next line with the `>` ?

Reply:
Prettier auto-formats it, can force it on new lines which is nicer